### PR TITLE
INGM-336 Fix all tests

### PR DESCRIPTION
**Pull request by pzinemanas**
### Description

This PR fixes all tests to have a stable situation in our test setup.

Fixes # INGM-336

### Type of change

- [x] Fix test of halls sensor
- [x] Add a fixture to load the configuration after each module is tested.
- [x] Fix issue with ethernet communication (INGK-687)

### Tests
- [x] Run tests more than 10 times and all tests passed.

### Documentation

Please update the documentation.

- [x] Update docstrings of every function, method or class that change.
- [x] Ue [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Build documentation locally to verify changes.
- [ ] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

File: requirements/requirements.txt
Comment (MartinALL91) [inline requirements/requirements.txt:1]: Maybe we should wait for this ingenialink branch to be merged.

File: tests/test_all_drive_tests.py
Comment (GullinBustin) [inline tests/test_all_drive_tests.py:142]: It looks like we are trying to hide a problem jaja

File: requirements/requirements.txt
Comment (pzinemanas) [reply requirements/requirements.txt:1]: Agreed. The idea was to revert this change before merging.

File: tests/test_all_drive_tests.py
Comment (pzinemanas) [reply tests/test_all_drive_tests.py]: Indeed. I decided to skip this test until we fix it.

**Review by MartinALL91 — APPROVED** [review]

**Review by WoodElv — APPROVED** [review]

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**
